.github/workflows/ac-check-changelog.yml

```diff
@@ -0,0 +1,33 @@
+name: AC Check Changelog
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - staging*
+
```

Comment:
Hey Isaac please add the ignore-paths section here similar to what we have in galaxy importer